### ISSUE TITLE
[Request] arm32 and arm64 support

### ISSUE THREAD

**Issue by pmatos** [Bogdanp/setup-racket#9]
What's required here to get arm32 and arm64 support?

**Comment by Bogdanp** [comment]
We'd need access to arm installers. If we had those, then all it would take would be to correctly map the URLs when `architecture` is `arm{32,64}` here:

https://github.com/Bogdanp/setup-racket/blob/11bb5ff19d1cd718876bc3cf8966a3fbce4354b7/src/common.ts#L32

We could also have a mode where it builds Racket from source but that would be very slow.

**Comment by pmatos** [comment]
Thanks - I will try to setup some arm, arm64 snapshots created on the `racket/racket` side.

**Comment by pmatos** [comment]
are you missing arm64 snapshots for this to work on arm64?

**Comment by Bogdanp** [comment]
Yup! I've added support for the arch and guessed at what the URLs will end up being, but there don't seem to be installers for arm64 on the Utah snapshot server yet so arm64 won't work yet.

**Comment by pmatos** [comment]
I will add snapshots for these in the next couple of days. I will keep you up-to-date.

**Comment by jessealama** [comment]
any traction here? I guess the snapshots should be available now.

**Comment by Bogdanp** [comment]
@jessealama it looks like only 32bit arm snapshots are available still: https://www.cs.utah.edu/plt/snapshots/

**Comment by Bogdanp** [comment]
Installers for Apple Silicon Macs are supported as of https://github.com/Bogdanp/setup-racket/pull/20/commits/94699d415b27bab4501b62f42f74b4eb8153d769 . GH Actions don't yet support Apple Silicon, though, so they aren't tested.

**Comment by dannypsnl** [comment]
I'm try to use 
```yaml
      - uses: Bogdanp/setup-racket@v1.7
        with:
          architecture: aarch64
```
get:
```
invalid arch 'aarch64'
```
what can I do here? Should I adjust `runs-on: ubuntu-latest` line?

**Comment by Bogdanp** [comment]
@dannypsnl I've just pushed a fix to the master branch for that problem. However, there aren't any arm64 linux installers on the Utah snapshot server yet so even with that fix in place, this still won't work. You can only currently use either `arm32` on Linux or `aarch64` on macOS. However, you need to bring your own GH Actions runners, because GitHub doesn't yet have arm runners.
